### PR TITLE
1010 reviewer suggestion

### LINKED ISSUE #1010: Update reviewer suggestion behaviour
Following discussions with the Editorial team, it is clear that all reviewer suggestions should be optional fields.

- By default, there should only be one 'Suggested Reviewer Name/Email' line of input fields
- All fields should be labelled (optional)
- Typing input into the first line should add a second line, typing input into the second line should add a third line etc.
- If a user has typed into into one field, the application should not let them continue until they have inputed information into the corresponsing second field (e.g. if a user types a name into Suggested Reviewer 1 name they will then need to input an email into Suggested Reviewer 1 email)
- Other validation should remain the same (e.g. no duplicated names or emails)

### LINKED ISSUE #1024: Correct typos in wizard
There are some typos throughout the app that need to be corrected:

- screen shot 2018-11-13 at 08 48 35

Should be 'Senior Editors'

- screen shot 2018-11-13 at 08 49 09

Should be 'Reviewing Editors'

- Final modal should read 'will be sent...' (currently reads 'will be send...')

### PR DESCRIPTION
**Pull request by Cooryd**
#### Background

- Removes the requirement to have three suggested reviewers making all optional (but maintaining the limit of six)

- Reduces the initial number of suggested reviewer inputs displayed from three to one

- Extends the EditorPage unit test to test the functionality of adding new inputs when the user fills in blank suggestedReviewer rows and remove trailing empty detail rows.

- Corrects some typos in the props passed to the People picker modal components.

#### Any relevant tickets

closes #1010
closes #1024

#### How has this been tested?
**Reviewers** ensure that these test requirements are also reviewed.

Please indicate the need for this change to be tested. If not please justify, if so then at what levels. 

- [x] Unit / Integration tests
- [ ] Browser tests
- [ ] End2end tests

#### UI Changes

- [ ] Has been reviewed against Designs
- [ ] Necessary updates to styleguide

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Will simplify https://github.com/elifesciences/elife-spectrum/pull/154/files#diff-2ba27ec3f3faa7e28b74c2b4718ac664R123 after this, if they are all optional they shouldn't be tested in end2end as it's not a feature involving multiple services.

File: app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js [hunk 1]
```diff
@@ -275,29 +275,31 @@ class EditorsPage extends React.Component {
...
+                    name={`suggestedReviewers.${index}.name`}
+                    onChange={this.handleSuggestedReviewersChanged}
+                  />
+                  <ValidatedField
+                    label={
+                      index < limits.suggestedReviewers.min
+                        ? `Reviewer ${index + 1} email`
+                        : `Reviewer ${index + 1} email (optional)`
+                    }
+                    name={`suggestedReviewers.${index}.email`}
```
Comment (giorgiosironi) [inline app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js]: names are preserved, for end2end testing to select fields using them

File: app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js [hunk 2]
```diff
@@ -275,29 +275,31 @@ class EditorsPage extends React.Component {
...
-                      ? `Reviewer ${index + 1} email`
-                      : `Reviewer ${index + 1} email (optional)`
-                  }
-                  name={`suggestedReviewers.${index}.email`}
-                  onChange={this.handleSuggestedReviewersChanged}
-                  type="email"
-                />
-              </TwoColumnLayout>
-            </Box>
+            <div data-test-id="suggestedReviewerInputGroup" key={index}>
```
Comment (SemraSh) [inline app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js]: Is there a reason not to use `React.Fragment` instead of `div` here?

Comment (Cooryd) [reply app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js]: However I suppose I could have an inner `React.Fragment` then run the selector as `wrapper.find('[data-test-id="suggestedReviewerInputGroup"] > React.Fragment')` that'd remove the need for the unnecessary `div`.

Comment (Cooryd) [reply app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js]: Managed to get the selector working without having to add a `div` or `React.Fragment` after reviewing. Thanks!

Comment (SemraSh) [reply app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js]: Ok sounds reasonable, thank you 👍

**Pull request merged**